src/main/java/com/mitchej123/hodgepodge/core/LoadingConfig.java

```diff
@@ -321,6 +325,15 @@ public LoadingConfig(File file) {
                         "If using Bukkit/Thermos, the CraftServer package.")
                 .getString();
 
+        renderDebug = config.get(
+                        "debug",
+                        "renderDebug",
+                        true,
+                        "Enable GL state debug hooks. Will not do anything useful unless mode is changed to nonzero.")
+                .getBoolean();
+        renderDebugMode = config.get(
```

Comment:
I don't think this is a good approach since it does not have any validation and can be provided any value, please wrap it to either verify input data or use text presentation instead.

Reply:
Good catch. Fixed now.